### ISSUE TITLE
Windows Qt client: NVDA screen reader reports misleading keyboard accelerators in Client -> Sound Configuration menu. Pressing the reported accelerators within the Sound Config menu causes TeamTalk to crash

### ISSUE THREAD

**Issue by ScottChesworth** [BearWare/TeamTalk5#1761]
### Description

Using TeamTalk 5.12 portable Qt client on Win10, NVDA 2022.4 prepends Alt to the keyboard accelerators in the Client -> Sound Configuration menu. Technically these reports are accurate if the user holds Alt for the entire journey through the menu structure, but that UX isn't standard. Usually, the topmost level accelerator is reported with Alt prepended (EG "Client  collapsed  Alt+l", then once inside a menu, the Alt is removed from reported accelerators. The more standardized behaviour already seems to be happening with JAWS.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

I expected to hear the actual keyboard accelerators, without Alt prepended. For example, "Sound Configuration  collapsed  o", or once the menu is expanded "Input Devices  collapsed  I".

### Actual behavior

We currently hear "Sound Configuration  collapsed  Alt+o". Expanding the Sound Config menu, the same behaviour of Alt being prepended applies to all of the options contained within it. In the Client menu, hitting Alt+O doesn't work, and once the Sound Configuration menu is expanded, hitting any of those accelerators as they are reported right now causes TeamTalk to crash.

### Steps to reproduce problem

1. Load TeamTalk 5.12 Qt and NVDA 2022.4.
2. Hit Alt+L to enter the Client menu.
3. DownArrow to Sound Configuration submenu, notice that the reported keyboard accelerator is Alt+O, which won't work if you've already let go of Alt. It should just report the letter O as the accelerator at this point, because hitting that does work.
4. Expand the sound Config menu, DownArrow through the choices to hear that their accelerators also have Alt prepended.
5. Now try pressing any of those Alt+accelerator combinations, Alt+E for "Enable Echo Cancellation" for example. TeamTalk will crash.

**Comment by amirmahdifard** [comment]
yes, i can confurm this! this is actually happening and the teamtalk will crash by reproduceing this issue!

**Comment by lbk2907** [comment]
@bear101 @CoBC 
It happend also on TeamTalk 5.13.0.5108 - Unreleased. Alt+E will crash, Alt+D will open "Record Conversations to Disk" dialog, Alt+O will bring the focus to the first menu in the sound configuration submenu and Alt+OtherKey will close the sound configuration submenu. I not sure if other user happend the same but at least happend for me.

**Comment by bear101** [comment]
@CoBC Should I close as "Cannot reproduce"?

**Comment by CoBC** [comment]
I'm able to reproduce this on all menu, but it's due to a Qt issue, so closing as qt-accessibility, reported to Qt here:
[QTBUG-96156](https://bugreports.qt.io/browse/QTBUG-96156)

**Comment by ScottChesworth** [comment]
Thanks for passing along the report to the proper place!
